lib/elixir/lib/macro.ex

```diff
@@ -1969,7 +1969,7 @@ defmodule Macro do
   def quoted_literal?(term), do: is_atom(term) or is_number(term) or is_binary(term)
 
   @doc """
-  Expands a `quoted_literal` with the given `
+  Expands a `quoted_literal` with the given ```
```

Comment:
This is still warning

Reply:
Fixed.

Reply:
Before fix:

<img width="722" alt="Desktop Screenshot 2022 08 08 - 21 53 44 45" src="https://user-images.githubusercontent.com/484530/183447575-7a2621bb-105f-4f63-bac8-01ac214a94ce.png">

After fixed:

<img width="729" alt="Desktop Screenshot 2022 08 08 - 21 52 18 85" src="https://user-images.githubusercontent.com/484530/183447613-45ff50c5-47f7-4286-9aa3-07ddadc4aac8.png">

Reply:
But it should `with the given AST` not backquote?